parcel-resolver-veda/index.d.ts

```diff
@@ -255,7 +264,7 @@ declare module 'veda' {
   export const getOverride: (key: PageOverrides) => VedaDatum<any> | undefined;
 
   export const getString: (variable: string) => {
-    one: string;
```

Comment:
I think the strings should always have a default. Is there a use case where this would be different?
If this is needed then `other` should also be `string | undefined`

Reply:
This is not a change of how the data is used, (also out of scope for this pr) more of an attempt to reflect what the current data structure is like to types. I read that anything that required singular/plurals needs to have `one`  + `others`, but anything that doesn't require singular/plurals should have others == every string should have `others` but only some require `one`. (ex. `storiesBanner`)

Reply:
Both properties will be the there as explained in the other comment.